docs/layouts/absolute-layout.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebarDepth: 0
+---
+
+# AbsoluteLayout
+
+The `<AbsoluteLayout>` container is the simplest layout container in NativeScript.
```

Comment:
please use documentation template to be consistent with other pages

Reply:
updated according to the template